app/assets/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,81 @@
+body {
+  @include font-feature-settings("kern", "liga", "pnum");
+  -webkit-font-smoothing: antialiased;
```

Comment:
Properties should be ordered color, font-family, font-size, -webkit-font-smoothing, line-height